internal/elixir/templates/template.Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Base image
+FROM elixir:{{.ElixirVer}} AS build
+
+# Set working directory
+WORKDIR /app
+
+# Set environment variables
+ENV MIX_ENV=prod
+ENV PORT=8080
+
+# Copy the project files to the working directory
+COPY . /app
```

Comment:
Better to not `COPY . /app` so soon – it invalidates the layer cache. https://dev.to/shmink/multi-stage-docker-and-elixir-releases-1ce2

Reply:
Done.